### PR TITLE
[EXPLORER] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

File: base/shell/explorer/lang/ro-RO.rc [hunk 1]
```diff
@@ -79,7 +79,7 @@ BEGIN
     AUTOCHECKBOX "&Menține bara de activități deasupra celorlalte ferestre", IDC_TASKBARPROP_ONTOP, 13, 71, 200, 10
     AUTOCHECKBOX "&Grupează butoanele similare din bara de activități", IDC_TASKBARPROP_GROUP, 13, 84, 200, 10
     AUTOCHECKBOX "Afișează bara „&Lansări rapide”", IDC_TASKBARPROP_SHOWQL, 13, 97, 200, 10
-    AUTOCHECKBOX "U&se small icons", IDC_TASKBARPROP_SMALLICONS, 13, 110, 200, 10
+    AUTOCHECKBOX "Folosește iconițe mici", IDC_TASKBARPROP_SMALLICONS, 13, 110, 200, 10
```
Comment (binarymaster) [inline base/shell/explorer/lang/ro-RO.rc:82]: Please assign a mnemonic key.

File: base/shell/explorer/lang/ro-RO.rc [hunk 2]
```diff
@@ -106,8 +106,8 @@ BEGIN
     LTEXT "Puteți menține ordonată zona de notificare prin ascunderea pictogramelor neaccesate recent.", IDC_STATIC, 13, 48, 223, 16
     AUTOCHECKBOX "Asc&unde pictogramele neaccesate", IDC_TASKBARPROP_HIDEICONS, 13, 70, 125, 10
     PUSHBUTTON "&Particularizare…", IDC_TASKBARPROP_ICONCUST, 188, 70, 50, 14
-    GROUPBOX "System icons", IDC_STATIC, 6, 99, 240, 56
-    LTEXT "Select which system icons to always show.", IDC_STATIC, 13, 113, 223, 8
+    GROUPBOX "Iconițe de sistem", IDC_STATIC, 6, 99, 240, 56
+    LTEXT "Selectați ce iconițe de sistem urmează să fie afișate mereu.", IDC_STATIC, 13, 113, 223, 8
     AUTOCHECKBOX "Afișează &ceas", IDC_TASKBARPROP_CLOCK, 13, 127, 80, 10
     AUTOCHECKBOX "Afișează s&ecunde", IDC_TASKBARPROP_SECONDS, 120, 127, 80, 10, WS_DISABLED
     AUTOCHECKBOX "&Desktop", IDC_TASKBARPROP_DESKTOP, 13, 140, 80, 10
```
Comment (binarymaster) [inline base/shell/explorer/lang/ro-RO.rc:113]: Left untranslated?

**Comment by AndreiMiloiu** [comment]
I fixed them. Sorry for this. I didn't pay attention.

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

File: base/shell/explorer/lang/ro-RO.rc [hunk 3]
```diff
@@ -79,7 +79,7 @@ BEGIN
     AUTOCHECKBOX "&Menține bara de activități deasupra celorlalte ferestre", IDC_TASKBARPROP_ONTOP, 13, 71, 200, 10
     AUTOCHECKBOX "&Grupează butoanele similare din bara de activități", IDC_TASKBARPROP_GROUP, 13, 84, 200, 10
     AUTOCHECKBOX "Afișează bara „&Lansări rapide”", IDC_TASKBARPROP_SHOWQL, 13, 97, 200, 10
-    AUTOCHECKBOX "&Folosește iconițe mici", IDC_TASKBARPROP_SMALLICONS, 13, 110, 200, 10
+    AUTOCHECKBOX "Folo&sește pictograme mici", IDC_TASKBARPROP_SMALLICONS, 13, 110, 240, 10
```
Comment (reactos-contributor) [inline base/shell/explorer/lang/ro-RO.rc:82]: Assignment of access key on "S" is not good in Romanian, as it gets needlessly painted very close to "Ș" (s with comma below), see [Romanian translation notes, pt. №17](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt#L41). Please change it to something else. The keys used in this dialog are "A", "G", "L", "M", and "X" (plus "F" and "N"), so given the text of this autocheckbox (and that "S", "Ș", "T", and "P" are not good candidates), the options that remain are "O", "E", "C", and "R".
Also, the access key on line №78 is overriding what should have been left for "Aplică". I suggest assigning something different for that checkbox too, like "U", or "D".

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

File: base/shell/explorer/lang/ro-RO.rc [hunk 4]
```diff
@@ -13,16 +13,16 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "&Modifică Data/Ora", ID_SHELL_CMD_ADJUST_DAT
-        MENUITEM "&Personalizare notificări…", ID_SHELL_CMD_CUST_NOTIF
+        MENUITEM "Modifică &Data/Ora", ID_SHELL_CMD_ADJUST_DAT
+        MENUITEM "P&ersonalizare notificări…", ID_SHELL_CMD_CUST_NOTIF
```
Comment (reactos-contributor) [inline base/shell/explorer/lang/ro-RO.rc:17]: "D" access key is already assigned two lines below.
"P" (the capital one) is fine as it doesn't have a "descender". See more on [Microsoft's guide on Selecting Access Keys](https://learn.microsoft.com/ro-ro/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design?redirectedfrom=MSDN#selecting-access-keys).

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

File: base/shell/explorer/lang/ro-RO.rc [hunk 5]
```diff
@@ -74,12 +74,12 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x1
...
-    AUTOCHECKBOX "Fi&xează bara de activități", IDC_TASKBARPROP_LOCK, 13, 45, 200, 10
-    AUTOCHECKBOX "&Auto-ascunde bara de activități", IDC_TASKBARPROP_HIDE, 13, 58, 200, 10
-    AUTOCHECKBOX "&Menține bara de activități deasupra celorlalte ferestre", IDC_TASKBARPROP_ONTOP, 13, 71, 200, 10
+    AUTOCHECKBOX "Fixea&ză bara de activități", IDC_TASKBARPROP_LOCK, 13, 45, 200, 10
+    AUTOCHECKBOX "Auto-ascun&de bara de activități", IDC_TASKBARPROP_HIDE, 13, 58, 200, 10
+    AUTOCHECKBOX "Menține &bara de activități deasupra celorlalte ferestre", IDC_TASKBARPROP_ONTOP, 13, 71, 200, 10
     AUTOCHECKBOX "&Grupează butoanele similare din bara de activități", IDC_TASKBARPROP_GROUP, 13, 84, 200, 10
-    AUTOCHECKBOX "Afișează bara „&Lansări rapide”", IDC_TASKBARPROP_SHOWQL, 13, 97, 200, 10
-    AUTOCHECKBOX "U&se small icons", IDC_TASKBARPROP_SMALLICONS, 13, 110, 200, 10
+    AUTOCHECKBOX "Afișează bara „La&nsări rapide”", IDC_TASKBARPROP_SHOWQL, 13, 97, 200, 10
```
Comment (reactos-contributor) [inline base/shell/explorer/lang/ro-RO.rc:81]: Please avoid assigning the "N" action key in dialog boxes, as it is assigned to "Anulează" button (of the dialog box that this form is in).

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

File: base/shell/explorer/lang/ro-RO.rc [hunk 6]
```diff
@@ -74,12 +74,12 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
     GROUPBOX "Aspectul barei de activități", IDC_STATIC, 6, 6, 240, 120
     CONTROL "", IDC_TASKBARPROP_TASKBARBITMAP, "Static", SS_BITMAP | SS_SUNKEN | SS_CENTERIMAGE, 12, 18, 229, 24
-    AUTOCHECKBOX "Fi&xează bara de activități", IDC_TASKBARPROP_LOCK, 13, 45, 200, 10
+    AUTOCHECKBOX "Fixează bara de acti&vități", IDC_TASKBARPROP_LOCK, 13, 45, 200, 10
     AUTOCHECKBOX "&Auto-ascunde bara de activități", IDC_TASKBARPROP_HIDE, 13, 58, 200, 10
```
Comment (reactos-contributor) [inline base/shell/explorer/lang/ro-RO.rc:78]: 1. I do not understand the need of changing the access key on line №78, as "A" was necessary for "Aplică" button of the dialog box. When it's about dialog boxes, please avoid assigning "F", "N", and (when "Apply" button is present) also "A" access keys to other controls.
2. (Less important) I do not quite understand the reason behind moving the access key from "X" to "V" on line №77. Neither "X" got reused elsewhere in this dialog box, nor "V" was a obviously much better option for this control.

**Review by reactos-contributor — CHANGES_REQUESTED** [review]
Other than the mentioned access key assignment, this looks good to me.

File: base/shell/explorer/lang/ro-RO.rc (hunk 6 above)
Comment (AndreiMiloiu) [reply base/shell/explorer/lang/ro-RO.rc]: I solved the problems. I still try to familiarize with the rules.

**Review by reactos-contributor — APPROVED** [review]

**Pull request merged**